client/src/components/about/Contact.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { FunctionComponent } from 'react';
+import { withTheme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import styled from 'styled-components';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+
+import { VerticalDivider } from '../divider/VerticalDivider';
+import ContactCardLogo from '../../assets/Contact_card_logo.png';
+
+interface ContactProps {
+  className?: string;
+}
+
+const UnstyledContact: FunctionComponent<ContactProps> = ({ className }) => {
+  return (
+    <Box className={className}>
+      <Container>
+        <Box className="contact" mt={10}>
+          <Box className="contact-title" mb={5}>
+            Contact us
+            <Box className="title-line" mt={2.5} />
+          </Box>
+
+          <Box display="flex" alignItems="flex-end" justifyContent="flex-end">
+            <Box className="contact-form" mr={10}>
+              <form>
+                <Box className="firstname-form" mb={10}>
+                  <div>First name</div>
+                  <div>
+                    <br />
+                  </div>
+                  <input type="text" className="form-control" id="firstname" placeholder="John Doe"></input>
+                </Box>
+
+                <Box className="email-form" mb={10}>
+                  <div>E-mail</div>
+                  <div>
+                    <br />
+                  </div>
+                  <input type="text" className="form-control" id="Email" placeholder="JohnDoe@KCSA.com"></input>
+                </Box>
+                <Box className="email-form" mb={10}>
+                  <div>Comments</div>
+                  <div>
+                    <br />
+                  </div>
+                  <input type="text" className="form-comments" id="comment" placeholder="Comment"></input>
+                </Box>
+                <Box className="button-form" mb={10}>
+                  <button type="submit" className="form-summit">
+                    submit
+                  </button>
```

Comment:
We already have `Button` component. Use that instead of raw `button`. 

Also fyi, this should actually trigger sending email to `koreancanadiansoccer@gmail.com`.

https://stackoverflow.com/questions/48818338/how-to-use-mailto-with-material-ui